src/CyBy/Draw/Internal/Graph.idr

```diff
@@ -604,6 +604,28 @@ newEdges t ps = do
   (a1,a2) <- ps
   (\(x,l) => (a1,x,l)) <$> neighboursAsPairs t a2
 
+inBond : {m : _} -> (n1,n2 : Fin m) -> Edge m CDBond -> Bool
+inBond n1 n2 bnd =
+  (n1 == bnd.node1 && n2 == bnd.node2) || (n2 == bnd.node1 && n1 == bnd.node2)
+
+-- if there is a template bond which would be deleted due to the node merging,
+-- create the bond between the corresponding origin molecule nodes
+-- TODO: the new type of the bond is Single, this should create the best(?)
```

Comment:
If this comment still holds, please open an issue where this can be discussed and remove the TODO. I hardly ever look at all those dangling TODOs in the code. You?